lib/response.js

```diff
@@ -58,16 +58,10 @@ Response = (function () {
     this._attachments.push(...attachments)
     this.meta('set_job_attachments', JSON.stringify(this._attachments))
   }
-  Response.prototype.job_ignore = function (status, data) {
-    this.meta('set_job_status', 'ignored')
-    this.meta('set_job_status_message', status)
-    if (data) {
-      this._respond(data)
-    }
-  }
-  Response.prototype.job_ignore = function (status, data) {
+  Response.prototype.job_ignore = function (status, reason, data) {
```

Comment:
Because this change isn't backwards compatible - can we get a list of integrations that need PRs on how they use the job_ignore method? Or can we make this backwards compatible?